examples/example-test.js

```diff
@@ -2,8 +2,9 @@ const Differencify = require('differencify');
 const globalConfig = require('./configs/global-config');
 const testConfig = require('./configs/test-config');
 
-const differencify = new Differencify(globalConfig);
+const differencify = new Differencify(globalConfig.default);
 
-differencify.test(testConfig).then((r) => {
-  console.log(r);
+differencify.test(testConfig.default).then((result) => {
+  console.log(result);
```

Comment:
`console.log(result)` here is a bit unclear. It's not clear it's a boolean or what to actually do with it. Something like `if (!result) throw new Error()` would be clearer, no?